dnd/config/main.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
In config file is where we make use on `ENV` variables. Something like this:
```
 host: process.env.HOST ,
  port: process.env.PORT ,
  dbpath: process.env.DB_PATH ,
  jwt_key: process.env.JWT 
```

Reply:
Care to elaborate? In the end wouldn't be inputting the strings anyway?